src/VisualStudio/Core/Def/ProjectSystem/MetadataReferences/VisualStudioMetadataReference.Snapshot.cs

```diff
@@ -110,7 +110,7 @@ protected override PortableExecutableReference WithPropertiesImpl(MetadataRefere
         private string GetDebuggerDisplay()
```

Comment:
I recommend reviewing this PR in a particular order.  First start with the [ITemporaryStorageServiceInternal](https://github.com/dotnet/roslyn/pull/73115/files#diff-feda3477ee65b7f6c1a71dec489d072c133470b17f86ff9e9d86587e058537de) change.